protos/flyteidl/admin/launch_plan.proto

```diff
@@ -93,8 +93,11 @@ message LaunchPlanSpec {
 
     AuthRole auth_role                    = 9;
 
-    // Indicates the runtime priority of the execution. 
+    // Indicates the runtime priority of the execution.
     core.QualityOfService quality_of_service   = 16;
+
+    // Prefix for where offloaded data from user workflows will be written (ie Blobs, Schema, query data, etc.)
+    string output_data_prefix            = 17;
```

Comment:
can we keep the naming consistent? call it raw_outputdata_prefix
In the docstring it can be, of the format
`s3://bucket/key` OR `s3://bucket/`

Reply:
sure but what's raw about it?